scripts/sdk-api-diff.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+usage="--------------------------------------------
+Description: Compares API differences between two module versions.
+
+Usage: sdk-api-diff <module name> <module A path> <module B path> [OPTIONS]
+
+Options:
+  --target: Target platform and version e.g. x86_64-apple-ios14.0-simulator or arm64-apple-ios14.0
+    (Default: x86_64-apple-ios14.0-simulator)
```

Comment:
Is it a good idea to default to the simulator target?
Maybe it doesn't matter.

Reply:
I was thinking that we mostly use x86_64 builds for unit tests etc. I found it the most convenient setting for now but we can also remove that default value and make that parameter required

Reply:
It seems like a sensible default to me